bundles/org.openhab.binding.sensibo/src/main/java/org/openhab/binding/sensibo/internal/model/SensiboSky.java

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sensibo.internal.model;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import javax.measure.Unit;
+import javax.measure.quantity.Temperature;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.smarthome.core.library.unit.ImperialUnits;
+import org.eclipse.smarthome.core.library.unit.SIUnits;
+import org.openhab.binding.sensibo.internal.dto.poddetails.Measurement;
+import org.openhab.binding.sensibo.internal.dto.poddetails.ModeCapability;
+import org.openhab.binding.sensibo.internal.dto.poddetails.PodDetails;
+import org.openhab.binding.sensibo.internal.dto.poddetails.Schedule;
+
+/**
+ * The {@link SensiboSky} represents a Sensibo Sky unit
+ *
+ * @author Arne Seime - Initial contribution
+ */
+public class SensiboSky extends Pod {
+
+    private final String macAddress;
+    private final String firmwareVersion;
+    private final String firmwareType;
+    private final String serialNumber;
+    private Unit<Temperature> temperatureUnit;
+    private final String originalTemperatureUnit;
+    private final String productModel;
+    private Boolean smartMode;
+    private AcState acState = null;
+    private Double temperature = null;
+    private Double humidity = null;
+    private boolean alive = false;
+    private final String roomName;
+    private Schedule[] schedules;
```

Comment:
This is never initialized in the class and private. What do you need it for?

Reply:
Good catch, should have been initialized. Needed for future schedule channels.